### ISSUE TITLE
I need an example of how to define custom loss function

### ISSUE THREAD

**Issue by menkaur** [SciSharp/Keras.NET#45]
I'm trying to load model which uses custom loss function using following code:

                Python.Runtime.PythonEngine.Initialize();
                var objects = new Dictionary<string,string>();
                objects["prediction_loss"] = @"
				   def prediction_loss(a,b):    
					   import tensorflow.keras.backend as K
					   scale=K.minimum(K.abs(a),K.abs(b))

					   distance=K.abs(a-b)

					   return K.minimum(distance,distance/scale)";
                var model=Keras.Models.Sequential.LoadModel(@"mymodel.h5", objects);

I've tried several variants, with using PythonEngine.Compile, PythonEngine.Eval,and PythonEngine.Exec, but the model load always fails.

What is the proper way of doing this?

**Comment by deepakkumar1984** [comment]
Here is the example you can refer: https://github.com/SciSharp/Keras.NET/blob/master/Examples/BasicSamples/ImplementCallback.cs

**Comment by deepakkumar1984** [comment]
Refer to https://github.com/SciSharp/Keras.NET/issues/13
